### PR TITLE
[gps] Update ublox RTK support and ground tool

### PR DESCRIPTION
**Pull request by fvantienen**
Add RTK tool for ublox GPS and update the RTK messages

### CODE REVIEW

**Review by gautierhattenberger — COMMENTED** [review]
Why not including the header `var/include/ubx_protocol.h` that is already defining the macros ? And if not, their is no reason to keep them in the header file. The header file itself is not really needed at all I would say.

**Comment by fvantienen** [comment]
I did not include `ubx_protocol.h` because that one has other includes and dependencies on functions like `ubx_send_bytes`. That would add a lot of unneeded defines and bypasses to make that work.

I chose to use a separate header file, to make the `c` file more readable and keep those defines separated. It is not needed, but makes it easier to read

File: sw/airborne/modules/gps/gps_ubx.c [hunk 1]
```diff
@@ -60,7 +60,7 @@
 #define GOT_CHECKSUM1 8
 
 #define RXM_RTCM_VERSION        0x02
-#define NAV_RELPOSNED_VERSION   0x00
+#define NAV_RELPOSNED_VERSION   0x01
```
Comment (gautierhattenberger) [inline sw/airborne/modules/gps/gps_ubx.c:63]: what is the implication of this modification ?

Comment (gautierhattenberger) [reply sw/airborne/modules/gps/gps_ubx.c:63]: yes, if you can keep compatibility it is better I think

**Review by gautierhattenberger — APPROVED** [review]

**Pull request merged**